bugbug/bug_features.py

```diff
@@ -300,7 +300,29 @@ def cleanup_hex(text):
 
 
 def cleanup_dll(text):
-    return re.sub(r'\w+(\.dll|\.so|\.dylib)\b', '__DLL_NAME__', text)
+    firefox_dlls = [
```

Comment:
Let's also make this a constant, defining it outside of the function (just before the function) and in upper-case (a Python convention for constants).

Reply:
The entire `inv_match` can be the constant (but rename it something like FIREFOX_DLLS_MATCH), this way we only calculate it once and not for every call of the function.